cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "baseUrl": "http://localhost:3000/",
+  "experimentalFetchPolyfill": true,
+  "firefoxGcInterval": {
+    "runMode": null,
+    "openMode": null
```

Comment:
would like to comment why this is necessary (https://github.com/cypress-io/cypress/issues/6813), but json can't have comments

Reply:
Yeah I tried it. Didn't work